### ISSUE TITLE
Error running script after clean install from github

### ISSUE THREAD

**Issue by schmistenew** [Ragin-LundF/telegraf_fritzbox_monitor#9]
Hi,
I'am having some errors running your script.

```
Apr 21 07:58:00 m telegraf[3933768]: 2023-04-21T05:58:00Z E! [inputs.exec] Error in plugin: exec: exit status 1 for command "python3 /opt/telegraf_fritzbox/telegraf_fritzbox.py -i x.x.x.x -u userid -p password": Traceback (most recent call last):
Apr 21 07:58:00 m telegraf[3933768]:   File "/opt/telegraf_fritzbox/telegraf_fritzbox.py", line 6, in <module>
Apr 21 07:58:00 m telegraf[3933768]:     from modules.configuration import Configuration
Apr 21 07:58:00 m telegraf[3933768]:   File "/opt/telegraf_fritzbox/modules/configuration.py", line 5, in <module>
Apr 21 07:58:00 m telegraf[3933768]:     import hiyapyco
Apr 21 07:58:00 m telegraf[3933768]:   File "/usr/local/lib/python3.9/dist-packages/hiyapyco/__init__.py", line 30, in <module>
Apr 21 07:58:00 m telegraf[3933768]:     from jinja2 import Environment, Undefined, DebugUndefined, StrictUndefined, TemplateError
Apr 21 07:58:00 m telegraf[3933768]:   File "/usr/local/lib/python3.9/dist-packages/jinja2/__init__.py", line 12, in <module>
Apr 21 07:58:00 m telegraf[3933768]:     from .environment import Environment
Apr 21 07:58:00 m telegraf[3933768]:   File "/usr/local/lib/python3.9/dist-packages/jinja2/environment.py", line 25, in <module>
Apr 21 07:58:00 m telegraf[3933768]:     from .defaults import BLOCK_END_STRING
Apr 21 07:58:00 m telegraf[3933768]:   File "/usr/local/lib/python3.9/dist-packages/jinja2/defaults.py", line 3, in <module>
Apr 21 07:58:00 m telegraf[3933768]:     from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
Apr 21 07:58:00 m telegraf[3933768]:   File "/usr/local/lib/python3.9/dist-packages/jinja2/filters.py", line 13, in <module>
Apr 21 07:58:00 m telegraf[3933768]:     from markupsafe import soft_unicode
Apr 21 07:58:00 m telegraf[3933768]: ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/usr/local/lib/python3.9/dist-packages/markupsafe/__init__.py)
```

Maybe you van have a look.
Thanks Stephan

**Comment by rezmer** [comment]
Got the same problem

**Comment by rezmer** [comment]
Fixed it by installing another version of markupsafe

pip install markupsafe==2.0.1

**Comment by Ragin-LundF** [comment]
Should be fixed with #11 in version 1.3.0. The YAML parser was exchanged that this issue should no longer exist.
I've tested with Python 3.10.
Please download the latest version and execute:

```bash
pip3 install -r requirements.txt
./install.sh
```
